grouper/fe/templates/user-requests.html

```diff
@@ -0,0 +1,66 @@
+{% extends "base.html" %}
+{% from 'macros/ui.html' import paginator, dropdown with context %}
+
+{% block heading %}
+    <a href="/Users">Users</a>
+{% endblock %}
+
+{% block subheading %}
+    Pending Requests
+{% endblock %}
+
+{% block headingbuttons %}
+    {{ dropdown("limit", limit, [100, 250, 500]) }}
+    {{ paginator(offset, limit, total) }}
+{% endblock %}
+
+
+{% block content %}
+    <div class="row"><div class="col-sm-10 col-sm-offset-1">
+        {% if total > 0 %}
+        <table class="table table-elist">
+            <thead>
+                    <th class="col-sm-1">Modify</th>
+                    <th class="col-sm-2">Requested</th>
+                    <th class="col-sm-2">Requester</th>
+                    <th class="col-sm-2">Status</th>
+                    <th class="col-sm-2">Requested At</th>
+                    <th class="col-sm-2">Expiration</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for request in requests %}
+                    <tr>
+                        <td>
+                            <a href="/groups/{{request.groupname}}/requests/{{request.id}}"
```

Comment:
This is certainly convenient, but it currently has the unfortunate side-effect that you don't end up back on the aggregated requests page after actioning a request.  Could you add a from argument so that the GroupRequestUpdate handler can redirect back to the appropriate page?